pkg/golinters/goanalysis/runner_action.go

```diff
@@ -98,6 +98,12 @@ func (act *action) waitUntilDependingAnalyzersWorked() {
 func (act *action) analyzeSafe() {
 	defer func() {
 		if p := recover(); p != nil {
+			// This line allows to display "hidden" panic with analyzers like buildssa.
```

Comment:
In this case we duplicate linter's panic in logs (1 and 3):
```
ERRO [linters_context/goanalysis] tparallel: panic during analysis: interface conversion: interface {} is nil, not *buildssa.SSA, goroutine 1068 [running]:

ERRO [linters_context/goanalysis] buildssa: panic during analysis: in github.com/Antonboom/golangci-vs-ent-generics/ent.withHooks$1

ERRO [runner] Panic: tparallel: package "ent" (isInitialPkg: true, needAnalyzeSource: true): interface conversion: interface {} is nil, not *buildssa.SSA: goroutine 1068 [running]:
```

maybe it's possible to introduce some condition for this?
e.g. hardcode on `buildssa.Analyzer.Name` 🙈

Reply:
buildssa is one but not the only one that can produce this kind of problem.
I assume the duplication in this context because I prefer duplication to hidden panic.

Reply:
I found a better solution!

Reply:
> when an action is not a root action.

It's exactly what I meant, but I didn't know about this flag.
you are the best, thanks!